libsplinter/src/node_id/store/diesel/mod.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Diesel based NodeIdStore
+
+mod models;
+mod operations;
+mod schema;
+use operations::{
+    get_node_id::NodeIdGetOperation, set_node_id::NodeIdSetOperation, NodeIdOperations,
+};
+
+use super::error::NodeIdStoreError;
+use super::NodeIdStore;
+use diesel::r2d2::{ConnectionManager, Pool};
+pub use models::NodeID;
```

Comment:
This `pub use` is unnecessary, as the model struct is an internal implementation detail.